docs/undergrad.yaml

```diff
@@ -336,11 +336,11 @@ Single Variable Real Analysis:
     inverse trigonometric functions: 'real.arcsin'
     inverse hyperbolic trigonometric functions: 'real.arsinh'
   Integration:
-    integral over a segment of piecewise continuous functions: ''
+    integral over a segment of piecewise continuous functions: 'interval_integral'
```

Comment:
I don't understand that edit. I would like to see a lemma saying that piecewise continuous functions are integrable on segments.

Reply:
Oh I see.